mlrun/projects/project.py

```diff
@@ -1936,6 +1936,7 @@ def run(
         local: bool = None,
         schedule: typing.Union[str, mlrun.api.schemas.ScheduleCronTrigger, bool] = None,
         timeout: int = None,
+        overwrite_schedule: bool = False,
```

Comment:
I think here is can be just `overwrite` or `replace` without schedule, as we are not only overwriting the crontab but also the whole workflow that is being run,